styles/hero.css

```diff
@@ -0,0 +1,156 @@
+.hero-wrapper {
+  background: #282828;
+  max-width: 1920px;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/img/Line-longer-2.svg"), url("/img/Vector\ 29.svg"),
+    url("/img/Vector\ 30.svg");
+  background-repeat: no-repeat, no-repeat, no-repeat;
+  background-position: 40% center, bottom left, bottom right;
+  gap: 10rem;
```

Comment:
Tasarımdaki sağ alttaki görseli doğru konumlandırmak için buraya `position: relative;` vermelisin. 
![image](https://github.com/archis-academy/june-frontend-eskool-2/assets/78505988/ac9b73fa-755d-47c9-9fdf-8a4a4aa36ac6)